src/Parser/LetStatement.hs

```diff
@@ -0,0 +1,12 @@
+module Parser.LetStatement (module Parser.LetStatement) where
```

Comment:
Let statement is going to have a whole class ??, Can you rename it as ParserFunction.hs

Reply:
Yeah, maybe is the best option.